pyke/lightcurve.py

```diff
@@ -182,20 +189,38 @@ def get_lightcurve(self, flux_type, centroid_type='MOM_CENTR'):
                                     campaign=self.campaign,
                                     quarter=self.quarter,
                                     mission=self.mission,
-                                    cadenceno=self.cadenceno)
+                                    cadenceno=self.cadenceno,
+                                    id = self.hdu[0].header['KEPLERID'])
         else:
             raise KeyError("{} is not a valid flux type. Available types are: {}".
                            format(flux_type, self._flux_types))
 
-    def _quality_mask(self, quality_bitmask):
+
+    def _quality_mask(self, bitmask, mask):
         """Returns a boolean mask which flags all good-quality cadences.
 
         Parameters
         ----------
         quality_bitmask : int
             Bitmask. See ref. [1], table 2-3.
         """
-        return (self.hdu[1].data['SAP_QUALITY'] & quality_bitmask) == 0
+        bitmask_dict = {'default': KeplerQualityFlags.DEFAULT_BITMASK,
+                'conservative': KeplerQualityFlags.CONSERVATIVE_BITMASK,
+                'hard': KeplerQualityFlags.QUALITY_ZERO_BITMASK,
+                'None':None,
+                None:None}
+        if bitmask == None:
```

Comment:
Do we need `bitmask == None`.  Let's define the default in the function call def?

Reply:
Fixed